NOTICE

```diff
@@ -0,0 +1,265 @@
+Petastorm includes derived work from CPython(https://github.com/python/cpython/blob/master/Lib/collections/__init__.py#L301). The derived work includes:
```

Comment:
This link points to a newer version of cpython. Its implementation of namedtuple is significantly different from the ones we have here.
This one is closer: https://github.com/python/cpython/blob/2.7/Lib/collections.py (but still not exactly the same). I wonder why are the differences?

Reply:
Oh right, I was referencing the master branch here. I will reference this one then (or another one even closer if I find it).

I'm not sure about the reason for the differences, I think it might have to do with the performances of the interpreter between string interpolation to define a class vs writing the method ahead.

Reply:
My implementation derives from this one: https://github.com/python/cpython/blob/3.6/Lib/collections/__init__.py
I will update the NOTICE to mention the 3.6 branch instead of the master one.